template/src/components/hello-world/index.js

```diff
@@ -1,12 +1,10 @@
-import { h, Component } from "preact";
-import "./style.scss";
+import { h } from "preact";
+import "./style.css";
 
-export default class App extends Component {
-  render(props) {
+export default function App(props) {
     return (
-      <div>
-        <h1 style={{ color: props.color }}>Hello, World!</h1>
-      </div>
+        <div>
```

Comment:
This seems to be the only file which is indented by 4 spaces now

Reply:
Very true. Sorry about that. Should be fixed now.

The only other file which may stand out is the `template.html` as it uses tabs for spacing. That was just copied straight from the CLI though, so I assume it's fine. Just as easy to fix too.